JitsiConference.js

```diff
@@ -3320,17 +3331,17 @@ JitsiConference.prototype.setE2EEKey = function(key) {
  *
  * @returns {void}
  */
-JitsiConference.prototype._setupSenderE2EEForTrack = function(track) {
-    const jvbPc = this.jvbJingleSession ? this.jvbJingleSession.peerconnection : null;
-
-    if (jvbPc && this._e2eeCtx) {
-        const sender = jvbPc.findSenderForTrack(track.track);
+JitsiConference.prototype._setupSenderE2EEForTrack = function(session, track) {
+    if (!this._e2eeCtx) {
+        return;
+    }
+    const pc = session.peerconnection;
```

Comment:
We ccould use geetActiveTPC` here as well instead of passsing the session around.

Reply:
but I want the peerconnection the track is received on, no matter what the current establishment status of the p2p ice connection is?

Reply:
That is taken care of by caller function already, isn't it?

Reply:
no, this is called from JitsiConference.prototype.onCallAccepted (line 1701) for p2p and JitsiConference.prototype._acceptJvbIncomingCall for the sfu call. In both cases we know exactly where this is originating

Reply:
Ah, gotcha.